docs/index.rst

```diff
@@ -0,0 +1,106 @@
+Shotgun Hiero Export API reference, |release|
```

Comment:
There is something weird about the doc build - not sure where to put the comment so adding it here - the TOC seems to contain the same items twice. I just realised that we have the same issue for the [nuke review](http://developer.shotgunsoftware.com/tk-nuke-quickreview/) docs but not for the [publish](http://developer.shotgunsoftware.com/tk-multi-publish2/). Could you investigate and potentially update the nuke review docs too?

Reply:
I'll ping you offline about this.

Reply:
We have an internal ticket to look at this more closely. It's not immediately obvious why the ToC is wonky.